packages/forma-36-website/src/pages/guidelines/date-and-time/index.mdx

```diff
@@ -0,0 +1,206 @@
+      <TableCell style={{ width: '25%' }}>Example</TableCell>
+      <TableCell style={{ width: '50%' }}>Notes</TableCell>
+    </TableRow>
+  </TableHead>
+  <TableBody>
+    <TableRow>
+      <TableCell>Relative date and time</TableCell>
+      <TableCell style={{ fontWeight: 700 }}>
+        Never
+        <br />
+        Less than an hour ago
+        <br />
+        13 days ago
+        <br />
+        3 months ago
+        <br />a year ago
+      </TableCell>
+      <TableCell></TableCell>
+    </TableRow>
+    <TableRow>
+      <TableCell>Absolute time</TableCell>
+      <TableCell style={{ fontWeight: 700 }}>8:00 am</TableCell>
+      <TableCell>
+        Time should be displayed in 12-hour format, with lowercase am or pm and
+        without periods, to increase scannability.
+      </TableCell>
+    </TableRow>
+    <TableRow>
+      <TableCell>Absolute dates</TableCell>
+      <TableCell style={{ fontWeight: 700 }}>13 Jan 2019</TableCell>
+      <TableCell>
+        Dates should be displayed in order of day-month-year, with month
+        abbreviated and without punctuation, to increase scannability and to
+        strike a compromise between different localized date formats.
+      </TableCell>
+    </TableRow>
+    <TableRow>
+      <TableCell>Absolute date and time</TableCell>
+      <TableCell style={{ fontWeight: 700 }}>8 Aug 2019 at 8:00 am</TableCell>
+      <TableCell>
+        Absolute date and time should be displayed without commas and as
+        human-readable as possible.
+      </TableCell>
+    </TableRow>
+    <TableRow>
+      <TableCell>Weekday</TableCell>
+      <TableCell style={{ fontWeight: 700 }}>
+        Mon, 12 Aug
+        <br />
+        Wed, 14 Aug at 11:00 pm
+      </TableCell>
+      <TableCell>
+        Weekdays should be used when users need to compare events that happen in
+        a daily context, always next to absolute date and time and separated
+        with a comma.
+      </TableCell>
+    </TableRow>
+  </TableBody>
+</Table>
+
+## Ranges
+
+To show a date or time range, use an **en dash** (Unicode: U+2013, html: `&ndash;`) with spaces around it.
+
+<Table className="f36-margin-bottom--xl" style={{ width: '100%' }}>
+  <TableHead>
+    <TableRow>
+      <TableCell style={{ width: '25%' }}>Type</TableCell>
+      <TableCell style={{ width: '25%' }}>Examples</TableCell>
+      <TableCell style={{ width: '50%' }}>Notes</TableCell>
+    </TableRow>
+  </TableHead>
+  <TableBody>
+    <TableRow>
+      <TableCell>Time</TableCell>
+      <TableCell style={{ fontWeight: 700 }}>8:00 am – 2:00 pm</TableCell>
+      <TableCell></TableCell>
+    </TableRow>
+    <TableRow>
+      <TableCell>Date</TableCell>
+      <TableCell style={{ fontWeight: 700 }}>
+        30 Mar – 30 Apr
+        <br />
+        8 Jul – 13 Aug 2019
+        <br />
+        13 Dec 2018 – 8 Aug 2019
+      </TableCell>
+      <TableCell>
+        Omit the year on the left side of the range if it's in the same year.
```

Comment:
If the range is within the same year, omit the year on the left side. If the year can be easily understood from the context, omit it completely.